### PR TITLE
Twnia2/user guide] 4. 套件與軟體

### PR DESCRIPTION
**Pull request by ting88510**
#214

### CODE REVIEW

File: TWNIA2/1. User Guides/4. 套件與軟體.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+# Packages and software
+
+The packages and software available for Taiwania2 (HPC CLI) are as follows:
+
+
+**【Software tools and Libraries】**
+
+- Mathematics software
+
+| Software name | Software version | 	Software platform |
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/4. 套件與軟體.md:19]: | Software name | Software version | 	Software platform | *4 >

| Name | Version| 	Platform |

File: TWNIA2/1. User Guides/4. 套件與軟體.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+| -------- | -------- | -------- |
+| [Matlab](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=c13edee6-ed57-4fed-9be1-5480b4a05302)       | For educational use     | UNIX, PC    |
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/4. 套件與軟體.md:21]: For educational use*6 > Non-industrial

File: TWNIA2/1. User Guides/4. 套件與軟體.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+| Software name | Software version | 	
+| -------- | -------- | 
+|  [TensorFlow](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=717a850d-4fb6-4033-a2f8-cd0d52cdf5b8)            | OpenSource edition     |     
+|  [PyTorch](https://iservice.nchc.org.tw/nchc_service/nchc_service_software_detail.php?diuu=66a576cb-56e9-4185-a328-a2ab806b38cf)               | OpenSource edition     |    
+
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+- How to use AI framework software? Please refer to [<ins>Create TWNIA2 container</ins>](https://man.twcc.ai/@twccdocs/howto-twnia2-create-sglrt-container-en) and [<ins>Multi-nodes parallel computing of container</ins>](https://man.twcc.ai/@twccdocs/howto-twnia2-run-parallel-job-container-en).
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/4. 套件與軟體.md:53]: container > containers

Comment (VigaWei) [reply TWNIA2/1. User Guides/4. 套件與軟體.md:53]: Multi-nodes parallel computing of container > Run parallel computing jobs with multi-nodes on containers

File: TWNIA2/1. User Guides/4. 套件與軟體.md [hunk 4]
```diff
@@ -1 +1,59 @@
...
+- If you want to use [<ins>NGC container</ins>](https://ngc.nvidia.com/catalog/containers), please contact <ins><a href="isupport@twcc.ai"></a>isupport@twcc.ai</ins>.
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/4. 套件與軟體.md:54]: container > containers

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
Revised, thanks!!

**Pull request merged**